Algorithms-Big-O-Notation.md

```diff
@@ -0,0 +1,54 @@
+# Big-O Notation
+
+In mathematics, Big-O notation is a symbolism used to describe and compare the limiting behavior of a function.
```

Comment:
> In mathematics, Big-O notation is a symbolism used to describe and compare the limiting behavior of a function.

Great opening, but maybe you would add a line or two explaining what you mean by _limiting behavior_.